boards/nxp/frdm_mcxn947/board.c

```diff
@@ -84,6 +84,15 @@ __ramfunc static void enable_cache64(void)
 }
 #endif
 
+static void unsecure_gpio(GPIO_Type * base)
+{
+	/* Enables CPU1 to access GPIO registers */
+	base->PCNS = 0xFFFFFFFFU; /* Pins can be configured in non-secure access */
+	base->PCNP = 0xFFFFFFFFU; /* Pins can be configured in non-privilege access */
+	base->ICNS = 0xFFFFFFFFU; /* Interrupts can be configured in non-secure access */
+	base->PCNS = 0xFFFFFFFFU; /* Interrupts can be configured in non-privilege access */
```

Comment:
this is the same register as the first line, is it intentional?

Reply:
Good catch @decsny .  No that was my error, not intentional.  And while confirming the proper register name, I realized I was setting many reserved bits.  Now updated to fix this.  Thank you